lib/msal-browser/docs/accounts.md

```diff
@@ -6,10 +6,32 @@
 
 The `@azure/msal-browser` library provides the following APIs to access cached accounts:
 
-* `getAllAccounts()`: returns all the accounts currently in the cache. An application must choose an account to acquire tokens silently.
-* `getAccountByHomeId()`: receives a `homeAccountId` string and returns the matching account from the cache.
-* `getAccountByLocalId()`: receives a `localAccountId` string and returns the matching account from the cache.
-* `getAccountByUsername()`: receives a `username` string and returns the matching account from the cache.
+-   `getAllAccounts()`: returns all the accounts currently in the cache. A filter can be passed in to narrow down the returned accounts. An application must choose an account to acquire tokens silently.
+-   `getAccountByFilter()`: returns the first cached account that matches the filter passed in. The order in which accounts are read from the cache is arbitrary and there is no guarantee that the first account in the filtered list will be the same for any two calls of `getAccountByFilter`.
+
+### Account Filter Object
+
+The following table presents the properties that can be used in the `AccountFilter` object that can be passed into the APIs listed above.
+
+> Note: A single account filter attribute is usually not guaranteed to uniquely identify a cached account object. Adding a combination of attributes that don't repeat together, such as `homeAccountId` + `localAccountId`, can help refine the search.
+
+| Property Name        | Description                                                                                                                                                                                  | Example                                                                   |
```

Comment:
Let's reference the typedocs here instead of writing it all out - less to maintain later :)

Reply:
I knew this was coming 😆